app/src/main/res/values/dimens.xml

```diff
@@ -206,9 +206,9 @@
     <dimen name="account_settings_device_name_min_height">48dp</dimen>
 
     <!-- Top sites  -->
-    <dimen name="top_sites_favicon_size">40dp</dimen>
-    <dimen name="top_sites_favicon_padding">4dp</dimen>
-    <dimen name="top_sites_item_size">64dp</dimen>
+    <dimen name="top_sites_favicon_size">60dp</dimen>
+    <dimen name="top_sites_favicon_padding">12dp</dimen>
```

Comment:
This is the padding to get the favicon to be 36x36.

<img width="88" alt="Screen Shot 2021-04-08 at 4 21 52 PM" src="https://user-images.githubusercontent.com/1190888/114091330-8cc9b100-9886-11eb-85c0-21d76eb38213.png">